lib/cache.js

```diff
@@ -8,4 +8,15 @@ const keyv = new Keyv({
 
 keyv.on('error', logger.error.bind(logger));
 
+async function fetch(key, callback, ttl = undefined) {
```

Comment:
I just realized: Isn't `ttl` going to be undefined here regardless of the ` = undefined`

```
> function a (x, y, z) { console.log(x, y, z)}
undefined
> a()
undefined undefined undefined
undefined
> a(1)
1 undefined undefined
undefined
```

Reply:
Yeah, but this just makes it more clear to the reader that the 3rd argument is optional and is ok to be undefined.